tests/test_utils/system_tests_class.py

```diff
@@ -232,3 +238,20 @@ def run_dag(self, dag_id: str, dag_folder: str = DEFAULT_DAG_FOLDER) -> None:
         except Exception:
             self._print_all_log_files()
             raise
+
+    @staticmethod
+    def create_temp_file(filename, dir_path="/tmp"):
```

Comment:
```suggestion
    def create_dummy_file(filename, dir_path="/tmp"):
```
Temporary files are files that are used for a short time. However, they can contain valuable content.